agent/runner.go

```diff
@@ -29,6 +29,7 @@ import (
 	"go.woodpecker-ci.org/woodpecker/v2/pipeline"
 	backend "go.woodpecker-ci.org/woodpecker/v2/pipeline/backend/types"
 	"go.woodpecker-ci.org/woodpecker/v2/pipeline/rpc"
+	"go.woodpecker-ci.org/woodpecker/v2/server/model"
```

Comment:
agent should not import server

int looks like  model.ExitCodeKilled should be moved into `pipeline/*` package somewhere

Reply:
Done